internal/ir/infra.go

```diff
@@ -0,0 +1,65 @@
+package ir
+
+import (
+	cfgv1a1 "github.com/envoyproxy/gateway/api/config/v1alpha1"
+)
+
+// Infra defines managed infrastructure.
+type Infra struct {
+	// Proxy defines managed proxy infrastructure.
+	Proxy *ProxyInfra
```

Comment:
So, will the networking details be called out at this level? To put this another way, if we're using an LB Service for exposing the proxy, will that be configured at this level or inside ProxyInfra?

Reply:
Inside `ProxyInfra`. The `Infra` type was created since the Infra Manager may manage other infra beyond the proxy infra, e.g. provision the [ratelimit](https://github.com/envoyproxy/ratelimit) infra.